### DISCUSSION TITLE
如何用Ventoy安装Deepin to go？

### DISCUSSION THREAD

**Discussion by Sunrongguo2008** [ventoy/Ventoy#1992] [Q&A] [answered]
如何用Ventoy安装Deepin to go？
不是用Ventoy给其他设备安装Deepin，而是实现类似Windows to go（WTG）的效果。
希望知情的人给出链接或指导！🙏

**Comment by ventoy** [comment, marked as answer]
这个？ https://www.ventoy.net/cn/plugin_vtoyboot.html

**Reply by Sunrongguo2008** [reply]
谢谢！希望网站提供搜索功能！
